scripts.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const field = document.querySelector('#field');
+const ball = document.querySelector('#ball');
+
+
+  field.addEventListener('click', function(event) {
+    const fieldCoords = this.getBoundingClientRect();
```

Comment:
you can destruct properties from `fieldCoords ` and `ball ` for better readability
But it is up to you